dataplane/src/nat/mod.rs

```diff
@@ -245,9 +245,9 @@ impl Nat {
     /// From the destination IP address contained in `net`, finds the
     /// destination PIF for the packet.
     #[tracing::instrument(level = "trace")]
-    fn find_dst_pif(&self, net: &Net) -> Option<&Pif> {
+    fn find_dst_pif(&self, dst_ip: &IpAddr) -> Option<&Pif> {
```

Comment:
I'm a bit confused now. Are we going to keep the pif abstractions? cc @mvachhar

Reply:
Not that I know of. Note that `find_dst_pif()` is not added in this PR. This code was written at a time when we considered keeping the PIF abstraction. We'll need to adjust it when we've finalised the data model, but this PR does not attempt to address that.